README.md

```diff
@@ -207,16 +207,34 @@ This project is licensed under the MIT License - see the [LICENSE](LICENSE) file
 ## API
 Account management exposes various endpoints that aid in managing accounts within the P&I service.
 
-- POST `/account/add/pi` - used to add a user to the P&I user database. Takes in a header of the email of the admin
-issuing
+`/account/add/azure` - used to add an account into the azure AAD. Takes in users name, email and role, and attempts to
+persist this into azure.  Takes in a header of the email of the admin issuing the request and a body of a list
+of users to add to Azure following the [Account model](#Account) without the `azureAccountId` as this
+is created by azure.
+
+`/account/add/pi` - used to add a user to the P&I user database. Takes in a header of the email of the admin issuing
+
 the request and a body of a list of users to add to the database following the [P&I User model](#piuser) without the
 `userId` as this is created by the service.
 
+## Models
```

Comment:
I think the merge conflicts gone a bit strange (I had the same issue). The endpoint descriptions need to be within the API section, and then the Models section should just have the JSON models in it, with titles for what each one is

Reply:
Good catch - I've amended this and added the REST calls